lib/presentation/popular_monuments/desktop/monument_details_view_desktop.dart

```diff
@@ -406,7 +406,7 @@ class _MonumentDetailsViewDesktopState
                         borderRadius: BorderRadius.circular(12.sp),
                         image: DecorationImage(
                           image: CachedNetworkImageProvider(
-                            widget.monument.images[4],
+                            widget.monument.images[3],
```

Comment:
I think there's a better solution than this, by default the app expects at least 6 images. The UI could be changed to have a carousel I guess? That way we wouldn't have to deal with the number of images being less than 6. The mobile UI implements a carousel for this, maybe you can just copy paste the implementation for desktop with some width/height changes